user-docs/pupil-capture.md

```diff
@@ -36,12 +36,71 @@ By default Pupil Capture will use Local USB as the capture source. If you have a
 - Pupil Mobile - select this option When using Pupil Capture with the Pupil Mobile android application.
 - Local USB - select this option if your Pupil Headset is connected to the machine running Pupil Capture. This is the default setting.
 
+### Pupil Detection
+
+Pupil's algorithms automatically detect the participant's pupil. With the 3d detection and mapping mode, Pupil uses a 3d model of the eye(s) that constantly updates based on observations of the eye. This enables the system to compensate for movements of the headset - slippage. To build up an initial model, you can just look around your field of view.
```

Comment:
Looking around definitively helps with the build up of the model. But we are working [on improving it](https://github.com/pupil-labs/pupil/issues/685). The documentation will be changed appropriately as soon as we publish the improvements.